src/EFCore.Relational/Query/SqlExpressions/InExpression.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.EntityFrameworkCore.Query.SqlExpressions
     public class InExpression : SqlExpression
     {
         /// <summary>
-        ///     Creates a new instance of the <see cref="InExpression" /> class.
+        ///     Creates a new instance of the <see cref="InExpression" /> class which represents a test_expression IN subquery.
```

Comment:
```suggestion
        ///     Creates a new instance of the <see cref="InExpression" /> class which represents a '<paramref name="item" /> IN subquery' expression.
```

Same for other overloads